utoipa-gen/tests/schema_derive_test.rs

```diff
@@ -4477,7 +4474,6 @@ fn derive_schema_with_unit_hashmap() {
                 "volumes": {
                     "additionalProperties": {
                         "additionalProperties": {
-                            "type": "object",
```

Comment:
I am not sure about this one.  To begin with I dont get `HashMap<(), ()>` because in JSON keys must be strings. https://stackoverflow.com/questions/9304528/why-json-allows-only-string-to-be-a-key

Reply:
That was to provide some sort of compatibility for https://github.com/juhaku/utoipa/issues/576 to mitigate a case where `additionalProperties` key and value is both defined as empty object `{}` as seen in docker's OpenAPI specification.

Reply:
But for compatibility it does not matter whether there actually is `type: object` or not because it is enough that `utoipa` parses and compiles the schema.